todo-mvc-kitchensink/src/widgets/TodoItem.ts

```diff
@@ -0,0 +1,114 @@
+import { v, w } from '@dojo/widget-core/d';
+import { theme, ThemeableMixin, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { Todo } from './App';
+import { DestroyButton } from './DestroyButton';
+import { Label } from './Label';
+import * as styles from './styles/TodoItem.css';
+import { Toggler } from './Toggler';
+
+interface TodoCardItemProperties extends ThemeableProperties {
+	type: 'list' | 'card';
+	todo: Todo;
+	editTodo: Function;
+	removeTodo: Function;
+	toggleTodo: Function;
+}
+
+@theme(styles)
+export default class TodoCardItem extends ThemeableMixin(WidgetBase)<TodoCardItemProperties> {
+	render() {
+		const { type } = this.properties;
+
+		if (type === 'list') {
+			return this._returnListItem();
```

Comment:
tiny, but think these may be better named as `_renderListItem` rather than `_returnListItem`, what you think?